### ISSUE TITLE
[HOLD for payment 2023-10-09] [$500]  [Distance] - The toggle attribution options disappear when they are hovered

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#27863]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open request money
2. Change the tab to distance
3. Click on toggle attribution on bottom right side of map
4. Hover over the toggle attribution options

## Expected Result:
The toggle attribution options should not hide when they are hovered

## Actual Result:
The toggle attribution options gets hidden when hovered on web and also hidden when clicked on mweb

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [x] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.72.6
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/5c051b8e-8107-4566-af53-857944045a9d

https://github.com/Expensify/App/assets/93399543/1974c22e-fe2f-483c-a992-2c37ae5e679d

https://github.com/Expensify/App/assets/93399543/e1e550b3-baca-4f81-b345-42b4cfa89185

**Expensify/Expensify Issue URL:**
**Issue reported by:** @jo-ui
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1693917802357489

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01ead96ebe1b100d51</li>
        <li>Upwork Job ID: 1704487524512796672</li>
        <li>Last Price Increase: 2023-09-20</li>
<li>Automatic offers: </li>
<ul>
<li>bernhardoj | Contributor | 26797717</li>
<li>jo-ui | Reporter | 26797720</li>
</ul></li>
    </ul>
</details>

**Comment by bojangabric** [comment]
Contributor details
Your Expensify account email: bojangabric1997@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~014c9f9349470cd9cb

**Comment by robertKozik** [comment]
Hi @bojangabric @bernhardoj! Thank you both for your proposals and insights on this issue. In my opinion, the solution presented by @bernhardoj is the better one. I believe it's better to utilize existing styles rather than directly referencing the component classes if possible.

Selected Proposal: https://github.com/Expensify/App/issues/27863#issuecomment-1727900933
contributor: @bernhardoj 

🎀 👀 🎀 C+ reviewed

**Comment by robertKozik** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [X]  [@robertKozik] The PR that introduced the bug has been identified. Link to the PR: **It's not covered on original PR with implementation, so no particular PR introduce this behaviour**
> * [X]  [@robertKozik] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: **N/A**
> * [X]  [@robertKozik] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: **N/A**
> * [X]  [@robertKozik] Determine if we should create a regression test for this bug. **I don't think we need regression test for this as it's purely visual, one time, bug**
> * [X]  [@robertKozik] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again. **N/A**

tldr; I don't think we need the regression tests for this issue - it's rather one-time change, and bug won't come back anymore

**Comment by zanyrenney** [comment]
PAYMENT SUMMARY

The pull request got merged within 3 working days of assignment, so this job is eligible for a 50% #urgency bonus 🎉

@bernhardoj requires payment [offer](https://www.upwork.com/nx/wm/offer/26797717) (Contributor) - PAID $500 + $250 URGENCY BONUS
@robertKozik does not require payment (Contractor) - OWED $500 + $250 URGENCY BONUS
@jo-ui requires payment [offer](https://www.upwork.com/nx/wm/offer/26797720) (Reporter) PAID $50

**Comment by zanyrenney** [comment]
@robertKozik please seek your payment from ND. This is now all complete from Upwork/GH.

**Comment by robertKozik** [comment]
@zanyrenney as a contractor (Software Mansion) I'm not eligible for payment
